README.md

```diff
@@ -21,13 +21,13 @@ sbt test
 sbt it:test
 ```
 
-## To view the RAML
+## To view the OAS
 
 To view documentation locally ensure the Individuals Business EOPS API is running, and run api-documentation-frontend:
 `./run_local_with_dependencies.sh`
 
 Then go to http://localhost:9680/api-documentation/docs/api/preview and use this port and version:
```

Comment:
could you just amens the url to read openapi instead please 
http://localhost:9680/api-documentation/docs/openapi/preview